apps/site/pages/en/blog/release/v23.0.0.md

```diff
@@ -18,6 +18,16 @@ We’re excited to announce the release of Node.js 23! Key highlights include:
 Node.js 23 will replace Node.js 22 as the ‘Current’ release line when Node.js 22 enters long-term support (LTS) later this month.
 According to the release schedule, Node.js 23 will remain the ‘Current’ release for the next six months, until April 2025.
 
+### `require(esm)` is now enabled by default
+
+Support for loading native ES modules using `require()` had been available on v20 and v22 under the command line flag `--experimental-require-module`. In v23, this feature is now enabled by default. This feature is still experimental, and we are looking for user feedback to make more final tweaks before fully stabilizing it. For this reason, on v23, when the Node.js instance encounters a native ES module in `require()` for the first time, it will emit an experimental warning.
+
+If there happens to be any regressions caused by this feature, users can report it to the Node.js issue tracker. Meanwhile this feature can also be disabled using `--no-experimental-require-module` as a workaround. We expect to test this feature using v23, find out any potential regressions and make adjustments accordingly, before backporting the unflagging to v22. This will likely happen in a later semver-minor release after the the LTS promotion at the end of October.
```

Comment:
```suggestion
Support for loading native ES modules using `require()` had been available on v20.x and v22.x under the command line flag `--experimental-require-module`. In v23.x, this feature is now enabled by default. This feature is still experimental, and we are looking for user feedback to make more final tweaks before fully stabilizing it. For this reason, on v23.x, when the Node.js instance encounters a native ES module in `require()` for the first time, it will emit an experimental warning.

If there happens to be any regressions caused by this feature, users can report it to the Node.js issue tracker. Meanwhile this feature can also be disabled using `--no-experimental-require-module` as a workaround. We expect to test this feature using v23.x, find out any potential regressions and make adjustments accordingly, before backporting the unflagging to v22.x. This will likely happen in a later semver-minor release after the the LTS promotion at the end of October.
```